ocs_ci/ocs/dr/dr_workload.py

```diff
@@ -136,19 +136,23 @@ def __init__(self, **kwargs):
         self.workload_pod_count = config.ENV_DATA["dr_workload_pod_count"]
         self.workload_pvc_count = config.ENV_DATA["dr_workload_pvc_count"]
 
-        # Name of the preferred primary cluster
+        self.dr_policy_name = config.ENV_DATA.get("dr_policy_name") or (
+            dr_helpers.get_all_drpolicy()[0]["metadata"]["name"]
```

Comment:
Would the first item always provide the expected dr policy name?

Reply:
If a user wants a specific DRPolicy to be used, then it can be specified in `conf/ocsci/dr_workload.yaml` ,
otherwise assuming there is no preference for any policy, we use the first policy from the policy list.